python/paddle/fluid/tests/unittests/test_parallel_executor_seresnext.py

```diff
@@ -178,6 +179,15 @@ def optimizer(learning_rate=0.01):
     return optimizer
 
 
+def init_data(batch_size=2):
+    np.random.seed(5)
+    img = np.random.random(size=[batch_size] + img_shape).astype(np.float32)
+    label = np.array(
```

Comment:
label.astype("int64") ?

Reply:
merge the `init_data` function and simple_nets.init_data ?

Reply:
> label.astype("int64") ?

The type is defined in line 187.

Reply:
> merge the init_data function and simple_nets.init_data ?

Done